### ISSUE TITLE
[Bug][SF][2.6.0][FlexibleColumnLayout][A11y][Expected reply Time: ASAP] Resizable column separator does not meet the minimum target size

### ISSUE THREAD

**Issue by wilson-xu-sap** [SAP/ui5-webcomponents#10669]
### Describe the bug

Resizable column separator does not meet the minimum target size

<img width="1542" alt="Image" src="https://github.com/user-attachments/assets/6987d15d-1f2b-436a-ab63-2c98d92ecb0b" />

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/layouts-floorplans-flexiblecolumnlayout--docs&args=disableResizing:!true

### Reproduction steps

Check the size of separator

### Expected Behaviour

The keyboard focus should be visible on the separator in all modes.

### UI5 Web Components for React Version

2.6.0

### UI5 Web Components Version

2.6.0

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by petyabegovska** [comment]
Hello @SAP/ui5-webcomponents-topic-p,

Please check why the size of the Grip "Button"(used for resizing) is not following the VD specification?
In OpenUI5 it's also a different size.

Best regards,
Petya

**Comment by wilson-xu-sap** [comment]
Since 2.6 has some breaking changes, could this be downported to version 2.5.x?

**Comment by wilson-xu-sap** [comment]
Hi @ilhan007 This is still not fixed.  https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/layouts-floorplans-flexiblecolumnlayout--docs&args=headerText:List%20with%20ListItemStandard;selectionMode:None;separators:All;growing:Button

<img width="1517" alt="Image" src="https://github.com/user-attachments/assets/efd5e0a1-a90d-4a5a-8ffb-a1e792eb79fa" />

**Comment by nnaydenow** [comment]
Hi @wilson-xu-sap,

The fix is part of our nightly release and part of the latest release candidate https://github.com/SAP/ui5-webcomponents/releases/tag/v2.8.0-rc.2) . The fix will be available with next release.
